packages/components/badge/src/Badge/README.mdx

```diff
@@ -26,69 +26,82 @@ import { Badge } from '@contentful/f36-badge';
 
 Used for the entity that has been changed
 
-```jsx file=../../examples/PrimaryBadgeExample.tsx
+```tsx file=../../examples/PrimaryBadgeExample.tsx
```

Comment:
Types are not allowed in the example files, otherwise, it will break the playground we use. 
Even though `tsx` lang type here won't break anything, I'd suggest rolling it back to `jsx` for the sake of consistency with other components examples.

I'd say we probably need to change all the example files to `.jsx` instead. But it's not this PR scope for sure.